<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>724</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(54, 69, 79);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_File_Name">
    <property name="geometry">
     <rect>
      <x>34</x>
      <y>360</y>
      <width>453</width>
      <height>43</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	color:rgb(255,255,255);
	border:1px solid rgb(255,255,255);
	border-radius:12px;
	min-width:451;
	max-height:41;
	max-width:451;
	min-height:41;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="text">
     <string>File Name</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_Audio_File_Duration">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>481</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:horizontal {
	height:10px;
	width:460px;
	background:white;
	border-radius:5px;
}
QSlider::handle:horizontal {
	background:rgb(52,60,64);
	width:20px;
	height:20px;
	margin:-7px -7px;
	border-radius:10px;
}</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_Volume">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>600</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:horizontal {
	height:10px;
	width:350px;
	background:white;
	border-radius:5px;
}
QSlider::handle:horizontal {
	background:rgb(52,60,64);
	width:20px;
	height:20px;
	margin:-7px -7px;
	border-radius:10px;
}

QPushButton:Hover {
	background-color:rgb(140,140,140);
}</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Volume">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>590</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-radius:24px;
	background-color:rgb(255,255,255);
	color:rgb(52,51,64);
	padding:0px;
}

QPushButton:Hover {
	background-color:rgb(140,140,140);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_Elapsed_Duration">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>460</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	border:rgb(255,255,255);
}</string>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Total_Duration">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>460</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	border:rgb(255,255,255);
}</string>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>490</y>
      <width>351</width>
      <height>82</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_SeekBack">
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border:none;
	border-radius:24px;
	background-color:rgb(255,255,255);
	color:rgb(52,51,64);
	min-width:50px;
	max-width:50px;
	min-height:50px;
	max-height:50px;
	padding:0px;
}

QPushButton:Hover {
	background-color:rgb(140,140,140);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_PlayPause">
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border:none;
	border-radius:40px;
	background-color:rgb(255,255,255);
	color:rgb(52,51,64);
	min-width:80px;
	max-width:80px;
	min-height:80px;
	max-height:80px;
	padding:0px;
}

QPushButton:Hover {
	background-color:rgb(140,140,140);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_SeekForward">
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border:none;
	border-radius:24px;
	background-color:rgb(255,255,255);
	color:rgb(52,51,64);
	min-width:50px;
	max-width:50px;
	min-height:50px;
	max-height:50px;
	padding:0px;
}

QPushButton:Hover {
	background-color:rgb(140,140,140);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_Album">
    <property name="geometry">
     <rect>
      <x>61</x>
      <y>40</y>
      <width>400</width>
      <height>300</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius: 24px;
	background:rgb(52,60,64);
	}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>519</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOpen">
    <property name="title">
     <string>Open</string>
    </property>
    <addaction name="actionOpen_Audio_File"/>
   </widget>
   <addaction name="menuOpen"/>
  </widget>
  <action name="actionOpen_Audio_File">
   <property name="text">
    <string>Open Audio File</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
